src/settings.rs

```diff
@@ -15,14 +16,35 @@ const DEFAULT_RETENTION: &str = "100d";
 const DEFAULT_MAX_OPEN_FILES: i32 = 8000;
 const DEFAULT_MAX_MB_OF_LEVEL_BASE: u64 = 512;
 const DEFAULT_NUM_OF_THREAD: i32 = 8;
-const DEFAULT_MAX_SUBCOMPACTIONS: u32 = 2;
+const DEFAULT_MAX_SUB_COMPACTIONS: u32 = 2;
+
+#[derive(Parser, Debug)]
+#[command(version = env!("CARGO_PKG_VERSION"))]
+pub struct Args {
+    /// Path to the local configuration TOML file.
+    #[arg(short, value_name = "CONFIG_PATH")]
+    pub config: Option<String>,
+
+    /// Path to the certificate file.
+    #[arg(long, value_name = "CERT_PATH")]
+    pub cert: String,
+
+    /// Path to the key file.
+    #[arg(long, value_name = "KEY_PATH")]
+    pub key: String,
+
+    /// Path to the CA certificate files.
+    #[arg(long = "ca_certs", value_name = "CA_CERT_PATH")]
```

Comment:
```suggestion
    #[arg(long, value_name = "CA_CERT_PATH")]
```

굳이 underscore가 사용되도록 할 필요는 없을 것 같습니다. 하이픈이면 될 것 같아요.